src/bitmessageqt/__init__.py

```diff
@@ -2465,13 +2830,13 @@ def click_actionSettings(self):
                     "MainWindow", "Your maximum download and upload rate must be numbers. Ignoring what you typed."))
             else:
                 set_rates(BMConfigParser().safeGetInt("bitmessagesettings", "maxdownloadrate"),
-                    BMConfigParser().safeGetInt("bitmessagesettings", "maxuploadrate"))
+                          BMConfigParser().safeGetInt("bitmessagesettings", "maxuploadrate"))
```

Comment:
This is not necessary

Reply:
pylint begs to differ.

Reply:
https://www.python.org/dev/peps/pep-0008/#indentation